packages/react/src/components/Rating/Rating.base.tsx

```diff
@@ -178,7 +182,8 @@ export const RatingBase: React.FunctionComponent<IRatingProps> = React.forwardRe
           {...(readOnly && {
             allowFocusRoot: true,
             disabled: true,
-            'aria-label': ariaLabel,
+            role: 'textbox',
```

Comment:
This doesn't seem right... Per the aria spec, "textbox" is for things that allow freeform text input, which this definitely doesn't.